content/docs/2.0/scalers/influxdb.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Influxdb"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on influxdb queries"
++++
+
+### Trigger Specification
+
+This specification describes the `influxdb` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      authToken: some-auth-token
+      organizationName: influx-org
+      authTokenFromEnv: INFLUXDB_AUTH_TOKEN
+      organizationNameFromEnv: INFLUXDB_ORG_NAME
+      serverURL: http://influxdb:8086
+      thresholdValue: '4'
+      query: |
+        from(bucket: "bucket-of-interest")
+        |> range(start: -12h)
+        |> filter(fn: (r) => r._measurement == "stat")
+```
+
+**Parameter list:**
+
+- The `authToken`/`authTokenFromEnv` field(s) holds the authentication token needed for the influxdb client to communicate with an associated server. `authTokenFromEnv` should hold the name of the environment variable in the target deployment that contains the actual authorization token value.
+- `organizationName`/`organizationNameFromEnv` field(s) holds another value needed by the client. `organizationNameFromEnv` has the same semantics of pulling a value from the target deployment as `authTokenFromEnv`.
+- `serverURL` holds the url value of the influxdb destination server.
+- `thresholdValue` is provided by the user. This value can vary from use case to use case depending on the data of interest.
+- `query` is the flux query that will yield the value for the scaler to compare the `thresholdValue` against.
+
+### Example
```

Comment:
Can you add a section on authentication parameters similar to https://github.com/kedacore/keda-docs/blob/master/content/docs/2.0/scalers/azure-service-bus.md#authentication-parameters please?

This typically describes the scenarios, in this case auth token, and if it supports TriggerAuthentication or not